mongo/integration/unified/client_entity.go

```diff
@@ -170,10 +164,10 @@ func (c *clientEntity) processStartedEvent(_ context.Context, evt *event.Command
 	if !c.getRecordEvents() {
 		return
 	}
-	if _, ok := c.observedEvents["commandStartedEvent"]; ok {
+	if _, ok := c.observedEvents[commandStartedEvent]; ok {
```

Comment:
Does it matter that this is now technically `c.observedEvents["CommandStartedEvent"]` (`C` is now capitalized)? I thought `observedEvents` was lower camelCase and `storedEvents` upper.

Reply:
You're correct that the `observeEvents` array in the YAML/JSON test files contains camel-cased events, but I don't think this matters. The purpose of `observeEvents` is to tell us which events should be recorded so that the test file can make assertions on the events after the test operations are executed. How we store them internally in the test runner is not relevant. For example, Java may have an enum with values like `COMMAND_STARTED_EVENT` while Python might have `command_started_event` because idiomatic variable naming differs in those languages.

Reply:
Yep that makes sense! Clever to use `ToLower()` to encode `observedEvents` and `storedEvents` with the same `monitoringEventType` despite their casing difference in JSON.